netmaker-docs/mod-html.sh

```diff
@@ -67,4 +67,8 @@ find . -type f -name "*.html" -exec sed -i 's-var json_loc = ""versions.json""-v
 find . -type f -name "*.html" -exec sed -i 's-</head>-<link rel="shortcut icon" href="#">\n</head>-' {} +
 find . -type f -name "*.html" -exec sed -i 's#</head>#<script async src="REPLACETHISTHING" referrerpolicy="strict-origin-when-cross-origin"></script>\n</head>#' {} +
 find . -type f -name "*.html" -exec sed -i """s#</head>#<script> (function(ss,ex){ window.ldfdr=window.ldfdr||function(){(ldfdr._q=ldfdr._q||[]).push([].slice.call(arguments));}; (function(d,s){ fs=d.getElementsByTagName(s)[0]; function ce(src){ var cs=d.createElement(s); cs.src=src; cs.async=1; fs.parentNode.insertBefore(cs,fs); }; ce('https://sc.lfeeder.com/lftracker_v1_'+ss+(ex?'_'+ex:'')+'.js'); })(document,'script'); })('p1e024BeABJ7GB6d'); </script>\n</head>#""" {} +
+find . -type f -name "*.html" -exec sed -i """s#</head>#<script>
+    !function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(\".\");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement(\"script\")).type=\"text/javascript\",p.async=!0,p.src=s.api_host.replace(\".i.posthog.com\",\"-assets.i.posthog.com\")+\"/static/array.js\",(r=t.getElementsByTagName(\"script\")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a=\"posthog\",u.people=u.people||[],u.toString=function(t){var e=\"posthog\";return\"posthog\"!==a&&(e+=\".\"+a),t||(e+=\" (stub)\"),e},u.people.toString=function(){return u.toString(1)+\".people (stub)\"},o=\"capture identify alias people.set people.set_once set_config register register_once unregister opt_out_capturing has_opted_out_capturing opt_in_capturing reset isFeatureEnabled onFeatureFlags getFeatureFlag getFeatureFlagPayload reloadFeatureFlags group updateEarlyAccessFeatureEnrollment getEarlyAccessFeatures getActiveMatchingSurveys getSurveys getNextSurveyStep onSessionId setPersonProperties\".split(\" \"),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
+    posthog.init('phc_1vEXhPOA1P7HP5jP2dVU9xDTUqXHAelmtravyZ1vvES',{api_host:'https://us.i.posthog.com', person_profiles: 'identified_only'})
```

Comment:
replace the key with `POSTHOG_KEY`.
The actual key will be referred from the secrets and will be replaced during deployment